### PR TITLE
Convert to Polymer 3.0

### PR DESCRIPTION
**Pull request by dpikhulya**
The element (with tests and the demo) was converted to the Polymer 3.0 format (using npm instead of bower, and JavaScript modules instead of HTML imports).

Conversion was made with [polymer-modulizer](https://github.com/Polymer/polymer-modulizer), and a subsequent correction of the auto-translated code.

### CODE REVIEW

File: spine-abstract-item.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    return 'option';
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    this._ensureAttribute('tabindex', '0');
+    this._ensureAttribute('role', this._getAriaRole());
+  }
+}
+
```
Comment (armiol) [inline spine-abstract-item.js]: This line is redundant.

File: spine-header-item.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+ * TeamDev PROPRIETARY and CONFIDENTIAL.
+ * Use is subject to license terms.
+ */
+
+import '@polymer/iron-flex-layout/iron-flex-layout.js';
+import { SpineAbstractItem } from './spine-abstract-item.js';
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+
+/**
+ * `spine-header-item` can be used inside `paper-listbox` among its children to display a
```
Comment (armiol) [inline spine-header-item.js:12]: Is this really true with Polymer 3 `paper-listbox`? Just a point to discuss later.

**Review by armiol — APPROVED** [review]
@dpikhulya LGTM. See my comments though.

File: spine-header-item.js (hunk 2 above)
Comment (dpikhulya) [reply spine-header-item.js:12]: Polymer 3.0 just changes the "packaging" aspects (npm & ES6 modules), but retains element APIs as is, so the scenarios of how this element is used remain the same.

**Pull request merged**